pkg/deploy/controller/deploymentconfig/controller.go

```diff
@@ -118,8 +119,13 @@ func (c *DeploymentConfigController) Handle(config *deployapi.DeploymentConfig)
 				copied.Annotations[deployapi.DeploymentCancelledAnnotation] = deployapi.DeploymentCancelledAnnotationValue
 				copied.Annotations[deployapi.DeploymentStatusReasonAnnotation] = deployapi.DeploymentCancelledNewerDeploymentExists
 
-				updatedDeployment, err := c.rn.ReplicationControllers(copied.Namespace).Update(copied)
-				if err != nil {
+				// Retry faster on conflicts
+				var updatedDeployment *kapi.ReplicationController
+				if err := kclient.RetryOnConflict(wait.Backoff{Steps: 5}, func() error {
+					var err error
+					updatedDeployment, err = c.rn.ReplicationControllers(copied.Namespace).Update(copied)
```

Comment:
huh?  Doesn't this just ensure that you fail 5 times?  If `copied`'s `resourceVersion` doesn't change, you'll just conflict over and over.